packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1065,9 +1065,10 @@ export class AccountHandler {
     await this.customs.check(request, email, 'accountStatusCheck');
 
     try {
-      const exist = await this.db.accountExists(email);
+      const account = await this.db.accountRecord(email);
       return {
-        exists: exist,
+        exists: !!account,
+        verifierSet: !!account && account.verifierSetAt > 0,
```

Comment:
If the unverified account has a subscription on it already, wouldn't we consider that to exist?

Reply:
Where do we make that call?  Not sure if I understand.  The only code making use of that new property is the front end code in this PR.

Reply:
> If the unverified account has a subscription on it already, wouldn't we consider that to exist?

Yea we should consider that account to still exisit. The code above would return the account in the case, so I think it would still be good?